### PR TITLE
Upgraded Mockito to Latest Package by replacing returnNullOnMissingStub to OnMissingStub.returnDefault

### LINKED ISSUE #1548: Upgrade the Mockito package to the latest version
**Describe the bug**

`returnNullOnMissingStub` is now deprecated in `Mockito` and needs to be migrated to their latest API.

**Mitigation Process**

- Remove `// ignore: ....` from the top of `test/helpers/test_helpers.dart`
- Fix all the lint warnings by migrating `returnNullOnMissingStub` to its latest alternative.

### PR DESCRIPTION
**Pull request by ALIRAZA17**
**What kind of change does this PR introduce?**
I have updated Mockito Package to its latest version by replacing the deprecated function of returnNullOnMissingStub to OnMissingStub.returnDefault. This is the latest enum being used if we want to return null. Other functions are deprecated.

**Issue Number:**

Fixes #1548 

**Did you add tests for your changes?**

No need Just updated the Package

**If relevant, did you update the documentation?**

No need.

**Summary**

The parameter being used in the Mockito Package was deprecated and the new version in the form of onMissingStub was available. Just replaced the one with the other one.

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by literalEval** [comment]
@ALIRAZA17 please generate new mocks.

File: talawa_lint/bin/custom_lint.dart [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// ignore_for_file: talawa_api_doc
+// ignore_for_file: talawa_good_doc_comments
```
Comment (palisadoes) [inline talawa_lint/bin/custom_lint.dart:2]: Why was this added? @literalEval please comment.

File: talawa_lint/lib/helpers.dart [hunk 2]
```diff
@@ -1,3 +1,6 @@
+// ignore_for_file: talawa_api_doc
+// ignore_for_file: talawa_good_doc_comments
```
Comment (palisadoes) [inline talawa_lint/lib/helpers.dart:2]: Why was this added? @literalEval please comment

File: talawa_lint/lib/talawa_api_doc.dart [hunk 3]
```diff
@@ -1,3 +1,6 @@
+// ignore_for_file: talawa_api_doc
+// ignore_for_file: talawa_good_doc_comments
```
Comment (palisadoes) [inline talawa_lint/lib/talawa_api_doc.dart:2]: Why was this added? @literalEval please comment

File: talawa_lint/lib/talawa_good_doc.dart [hunk 4]
```diff
@@ -1,4 +1,6 @@
 // ignore_for_file: implementation_imports
+// ignore_for_file: talawa_api_doc
+// ignore_for_file: talawa_good_doc_comments
```
Comment (palisadoes) [inline talawa_lint/lib/talawa_good_doc.dart:3]: Why was this added? @literalEval please comment

File: talawa_lint/lib/talawa_lint.dart [hunk 5]
```diff
@@ -1,3 +1,6 @@
+// ignore_for_file: talawa_api_doc
+// ignore_for_file: talawa_good_doc_comments
```
Comment (palisadoes) [inline talawa_lint/lib/talawa_lint.dart:2]: Why was this added? @literalEval please comment

File: talawa_lint/lib/talawa_lint_rules.dart [hunk 6]
```diff
@@ -1,3 +1,6 @@
+// ignore_for_file: talawa_api_doc
+// ignore_for_file: talawa_good_doc_comments
```
Comment (palisadoes) [inline talawa_lint/lib/talawa_lint_rules.dart:2]: Why was this added? @literalEval please comment

File: test/helpers/test_helpers.dart [hunk 7]
```diff
@@ -1,8 +1,6 @@
 // ignore_for_file: talawa_api_doc
 // ignore_for_file: talawa_good_doc_comments
 
-// ignore_for_file: deprecated_member_use
```
Comment (palisadoes) [inline test/helpers/test_helpers.dart:4]: Why was this removed? @literalEval please comment

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please answer comments on edits

File: test/helpers/test_helpers.dart (hunk 7 above)
Comment (literalEval) [reply test/helpers/test_helpers.dart:4]: Because this is the deprecation that @ALIRAZA17 is trying to fix.

**Comment by literalEval** [comment]
@palisadoes for all of the changes you have mentioned other than the deprecation one, the answer is that the changes are unnecessary and I have informed @ALIRAZA17 to revert them.

File: test/helpers/test_helpers.dart (hunk 7 above)
Comment (ALIRAZA17) [reply test/helpers/test_helpers.dart:4]: Yes this is the deprecated version we are using and I have updated it to the new one
